pkg/apis/sources/v1alpha2/apiserver_lifecycle_test.go

```diff
@@ -49,7 +49,8 @@ func TestApiServerSourceGetGroupVersionKind(t *testing.T) {
 }
 
 func TestApiServerSourceStatusIsReady(t *testing.T) {
-	sink, _ := apis.ParseURL("uri://example")
+	sink := apis.HTTP("example")
+	sink.Scheme = "uri"
```

Comment:
We set the scheme to `http` and then reset it to `uri`.
I think it's better to leave as it was before because the scheme isn't HTTP.

Reply:
Initially I thought from the description that we should use HTTP everywhere but it is not the case. I will update it.